static/app/views/settings/projectProguard/associations.tsx

```diff
@@ -0,0 +1,131 @@
+import {Link} from 'react-router';
+import styled from '@emotion/styled';
+
+import ClippedBox from 'sentry/components/clippedBox';
+import {CopyToClipboardButton} from 'sentry/components/copyToClipboardButton';
+import {Hovercard} from 'sentry/components/hovercard';
+import Placeholder from 'sentry/components/placeholder';
+import TextOverflow from 'sentry/components/textOverflow';
+import {t, tn} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import useOrganization from 'sentry/utils/useOrganization';
+import {ProguardMappingAssociation} from 'sentry/views/settings/projectProguard';
+
+function ProguardAssociationsBody({
+  associations,
+}: {
+  associations: ProguardMappingAssociation;
+}) {
+  const organization = useOrganization();
+
+  return (
+    <ClippedBoxWithoutPadding
+      clipHeight={210}
+      btnText={t(
+        '+ %s more',
+        associations.releases.length - associations.releases.slice(0, 4).length
+      )}
+      buttonProps={{
+        priority: 'default',
+        borderless: true,
+      }}
+    >
+      <NumericList>
```

Comment:
we usually use the component https://github.com/getsentry/sentry/tree/master/static/app/components/list 😉